platform/marvell-armhf/sonic-platform-et6448m/entropy.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python
+import fcntl, struct
+import time
+
+RNDADDENTROPY=0x40085203
+
+def avail():
+  with open("/proc/sys/kernel/random/entropy_avail", mode='r') as avail:
+      return int(avail.read())
+
+while 1:
+    while avail() < 2048:
+        with open('/dev/urandom', 'rb') as urnd, open("/dev/random", mode='wb') as rnd:
+            d = urnd.read(512)
+            t = struct.pack('ii', 4 * len(d), len(d)) + d
+            fcntl.ioctl(rnd, RNDADDENTROPY, t)
+    time.sleep(30)
```

Comment:
>time.sleep(30) [](start = 4, length = 14)

why sleep?

Reply:
In A385 CPU arch, random number generator is not supported in CPU soc. During bootup, systemd requires lot of random seeds, so this has been handled in SW by providing entropy from pseudo random generator. Added sleep 30 to not to hog the CPU